src/lumigo_tracer/parsers/utils.py

```diff
@@ -274,3 +274,15 @@ def should_scrub_domain(url: str) -> bool:
             if re.match(regex, url, re.IGNORECASE):
                 return True
     return False
+
+
+def str_to_list(val: str):
+    if val:
```

Comment:
add try/except!
It runs in global time, and if there will be error- then it will crush the customer's lambda